core/resharding.go

```diff
@@ -134,12 +140,13 @@ func GetShardingStateFromBlockChain(bc *BlockChain, epoch uint64) *ShardingState
 // CalculateNewShardState get sharding state from previous epoch and calculate sharding state for new epoch
 // TODO: currently, we just mock everything
 func CalculateNewShardState(bc *BlockChain, epoch uint64, stakeInfo *map[common.Address]*structs.StakeInfo) types.ShardState {
-	if epoch == FirstEpoch {
-		return getInitShardState(3, 10)
+	if epoch == GenesisEpoch {
+		return GetInitShardState()
```

Comment:
how do you consolidate this hard-code config together with the params fed to deploy.sh?

if the config def to deploy.sh has 1 beacon chain with 20 nodes this may be not correct

Reply:
Yea, for now, I am keeping them in sync.

Reply:
that's fair enough. please add TODO